ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -631,6 +634,8 @@ objects:
           sed -n '/tasks/q;p' "${ASSETS_DIR}/network.yaml" > "${ASSETS_DIR}/down-containers.yaml"
           fi
 
+          BOOTSTRAP_FIP_ID="$(create_bootstrap_fip)
```

Comment:
`create_bootstrap_fip` function is returning the IP, and not the ID. You should change the variable name.